package.json

```diff
@@ -1,86 +1,38 @@
-    "test:cli": "node cli.js ./test/docs/dummy.yml @asyncapi/html-template -o test/output --force-write --debug && test -e test/output/index.html",
-    "test:cleanup": "rimraf \"test/temp\"",
-    "docs": "jsdoc2md --partial docs/jsdoc2md-handlebars/custom-sig-name.hbs docs/jsdoc2md-handlebars/main.hbs docs/jsdoc2md-handlebars/docs.hbs docs/jsdoc2md-handlebars/header.hbs docs/jsdoc2md-handlebars/defaultvalue.hbs docs/jsdoc2md-handlebars/link.hbs docs/jsdoc2md-handlebars/params-table.hbs --files lib/generator.js > docs/api.md",
-    "docker:build": "docker build -t asyncapi/generator:latest .",
-    "lint": "eslint --max-warnings 0 --config .eslintrc .",
-    "lint:tpl:validator": "eslint --fix --config .eslintrc .github/templates-list-validator",
-    "generate:readme:toc": "markdown-toc -i README.md",
-    "generate:assets": "npm run docs && npm run generate:readme:toc",
-    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
-  },
-  "preferGlobal": true,
-  "bugs": {
-    "url": "https://github.com/asyncapi/generator/issues"
+    "build": "turbo build",
+    "dev": "turbo dev",
+    "test": "turbo run test",
+    "lint": "turbo lint",
+    "format": "prettier --write \"**/*.{ts,tsx,md}\"",
+    "generator:test": "turbo run test --filter=@asyncapi/generator",
+    "generator:test:dev": "turbo run test:dev",
+    "generator:test:unit": "turbo run test:unit  --filter=@asyncapi/generator",
+    "generator:test:integration": "turbo run test:integration --filter=@asyncapi/generator",
+    "generator:test:cli": "turbo run test:cli --filter=@asyncapi/generator",
+    "generator:test:cleanup": "turbo run test:cleanup --filter=@asyncapi/generator",
+    "generator:docs": "turbo run docs --filter=@asyncapi/generator",
+    "generator:docker:build": "turbo run docker:build --filter=@asyncapi/generator",
+    "generator:lint": "turbo run lint --filter=@asyncapi/generator",
+    "generator-filters:test": "turbo run test --filter=@asyncapi/generator-filters",
+    "generator-filters:docs": "turbo run docs --filter=@asyncapi/generator-filters",
+    "generator-filters:release": "turbo run release --filter=@asyncapi/generator-filters",
+    "generator-filters:generate:assets": "turbo run generate:assets --filter=@asyncapi/generator-filters",
+    "generator-filters:lint:fix": "turbo run lint:fix --filter=@asyncapi/generator-filters",
+    "generator-filters:bump:version": "turbo run bump:version --filter=@asyncapi/generator-filters"
   },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/asyncapi/generator.git"
-  },
-  "keywords": [
-    "asyncapi",
-    "documentation",
-    "generator",
-    "markdown",
-    "codegen"
-  ],
-  "author": "Fran Mendez <fmvilas@gmail.com>",
-  "publishConfig": {
-    "access": "public"
+  "devDependencies": {
+    "prettier": "^3.2.5",
+    "turbo": "latest"
```

Comment:
should we really point to latest? isn't it risky and error prone?

Reply:
made it to the current latest version.